packaging/nnstreamer.spec

```diff
@@ -42,6 +42,11 @@
 %define		vivante_support 0
 %endif
 
+# Flatbuffers-dev package is necessary to build tensorflow-lite
+%if 0%{?tensorflow_lite_support}
+%define		flatbuf_support 1
```

Comment:
Line 9 also defines `flatbuf_support 1`. How about using if/else statements like other `*_support` variables.

Reply:
NO. tensorflow-dev should get flatbuf-dev automatically. nnstreamer.spec should never take care of such secondary dependencies.

Reply:
Yes, makes sense.

Reply:
Thank you.
I addded `flatbuffers-devel` dependency to `tensorflow-lite-devel`, I close this PR.
Please review this : https://review.tizen.org/gerrit/#/c/platform/upstream/tensorflow/+/236267/